packages/core/src/util/MaxXmlRequest.ts

```diff
@@ -251,9 +251,12 @@ class MaxXmlRequest {
    * @param ontimeout Optional function to execute on timeout.
    */
   send(
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type -- require a generic function type
     onload: Function | null = null,
```

Comment:
"Onload" specific signature can be inferred because it is used in this method .

This applies to onLoad parameters of all methods and functions of this file.